Firestore/core/test/firebase/firestore/model/field_value_test.cc

```diff
@@ -189,15 +189,14 @@ TEST(FieldValue, ArrayType) {
 }
 
 TEST(FieldValue, ObjectType) {
-  const FieldValue empty =
-      FieldValue::ObjectValue(std::map<std::string, FieldValue>{});
-  std::map<std::string, FieldValue> object{{"null", FieldValue::NullValue()},
-                                           {"true", FieldValue::TrueValue()},
-                                           {"false", FieldValue::FalseValue()}};
+  const FieldValue empty = FieldValue::ObjectValue(FieldValue::ObjectT{});
```

Comment:
Optional: just `FieldValue::ObjectValue({})` should work (applies to line 456 as well). Feel free to leave as is if you prefer to be more explicit.

Reply:
Done.